### PR TITLE
redirect root to swagger url

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

**Comment by garbas** [comment]
:+1: #

File: lib/releng_common/releng_common/__init__.py [hunk 1]
```diff
@@ -70,7 +70,12 @@ def create_app(name, extensions=[], config=None, debug=False, debug_src=None,
         if hasattr(app, 'log'):
             app.log.debug('extension `%s` configured.' % extension_name)
 
-    def get_run_options():
+    if redirect_root_to_api:
+        app.add_url_rule("/",
+                         "root",
+                         lambda: redirect(app.api._Api__api.swagger_url))
```
Comment (rail) [inline lib/releng_common/releng_common/__init__.py]: Ouch. Looks like you used a "private" attribute here. Do we really need to hide this rule here and make it implicit?

Comment (garbas) [inline lib/releng_common/releng_common/__init__.py]: i can also assign, ``swagger_url`` to ``app.api.url`` in ``class Api:``

**Comment by garbas** [comment]
closing this PR and i merged this changes on top of #85

**Pull request closed without merging**